app/views/user_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Sample App</h1>
+
+<p>Hi <%= @user.name %>,</p>
+
+<p>
+  Welcome to the Sample App! Click on the link below to activate your account:
+</p>
+
+<%= link_to "Activate", edit_account_activation_url(@user.activation_token,
```

Comment:
Khi đặt một liên kết trong trang web của mình,  domain của route là không cần thiết, vì vậy chỉ cần sử dụng một phần đường dẫn URL là _path. 
Mặt khác, nếu URL được được gọi tới bên ngoài của trang web , ví dụ email hoặc RSS feed, toàn bộ URL là cần thiết, vì vậy sử dụng  _url.